docs/internals/optimizer.rst

```diff
@@ -1216,6 +1216,11 @@ we can run the optimizer on this specialized function. If it
 results in heavy gains, the specialized function is kept,
 otherwise the original function is used instead.
 
+Function Hoister and ExpressionSplitter are recommended as prerequisites since they make the step
```

Comment:
```suggestion
FunctionHoister and ExpressionSplitter are recommended as prerequisites since they make the step
```

Reply:
Fixed.

These docs are pretty inconsistent about it though. You'll find all the possible forms (`FunctionHoister`, `Function Hoister` and `function hoister`) all over the place :)